.github/workflows/ci.yml

```diff
@@ -240,6 +240,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: seqan3
+          fetch-depth: 2
```

Comment:
This should fix the warning about not finding commit SHA (because github actions are run in detached HEAD), but the bash uploader still emitts the warning